packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -60,6 +68,16 @@ interface MongoNavInterface {
    * By default the value is set to `production`
    */
   mode?: Mode;
+
+  /**
+   * Function that is passed an error code, so that consuming application can handle fetch failures
+   */
+  onError?: (code: string) => void;
```

Comment:
This should follow our object-like enum pattern we have, where we accept only the values of all the keys of the object, but export the object itself so downstream users can use the object directly or just the string.

Reply:
👍